scanomatic/data_processing/project.py

```diff
@@ -32,19 +34,23 @@ def path_has_saved_project_state(
         unpickle_with_unpickler(
             np.load,
             os.path.join(directory_path,  _p.phenotypes_input_data),
+            allow_pickle=True,
         )
         unpickle_with_unpickler(
             np.load,
             os.path.join(directory_path, _p.phenotype_times),
+            allow_pickle=True,
         )
         unpickle_with_unpickler(
             np.load,
             os.path.join(directory_path, _p.phenotypes_input_smooth),
+            allow_pickle=True,
         )
-        unpickle_with_unpickler(
-            np.load,
+        with open(
             os.path.join(directory_path, _p.phenotypes_extraction_params),
-        )
+            'r',
+        ) as fp:
+            json.load(fp)
```

Comment:
this ought to be jsonizer and not just json

Reply:
Agreed, but see above.

Reply:
fixed